pymc3/distributions/transforms.py

```diff
@@ -47,46 +57,118 @@ def __init__(self, dist, transform, *args, **kwargs):
                 testval, dist.defaults, 
                 *args, **kwargs)
 
-
     def logp(self, x):
         return self.dist.logp(self.transform_used.backward(x)) + self.transform_used.jacobian_det(x)
 
 transform = Transform
 
+class Log(ElemwiseTransform):
+    name = "log"
+    def __init__(self): 
+        pass
+
+    def backward(self, x):
+        return exp(x)
 
-logtransform = transform("log", log, exp, idfn)
+    def forward(self, x):
+        return t.log(x)
 
+log = Log()
+logtransform = log
```

Comment:
This is for backwards compat? Maybe it's better to be ruthless here, especially early on.